### DISCUSSION TITLE
Adding ways to present ALL students to projects

### DISCUSSION THREAD

**Discussion by Typiqally** [xl3ehindTim/delta-site#59] [Ideas]
New design should be created to be able to show all students on projects they worked hard for. Now a picture is required, which is a massive shortcoming, as these pictures can not always be taken and will also differ in styling and quality when asking for different photographers.

One idea could be to have placeholder images for students who haven't had their picture taken. Or to generate different designs which could accommodate these students.

**Comment by xl3ehindTim** [comment]
I agree with an anonymous image in the same style as the current photos.
